packages/gen-ai-hub/src/orchestration/orchestration-client.test.ts

```diff
@@ -28,8 +24,7 @@ describe('GenAiHubClient', () => {
   });
 
   it('calls chatCompletion with minimum configuration', async () => {
-    const request: GenAiHubCompletionParameters = {
-      deploymentConfiguration,
+    const request = {
```

Comment:
Follow up for making the constants usable with orchestration: https://github.tools.sap/AI/gen-ai-hub-sdk-js-backlog/issues/91
Follow up for adding convenience for `model_params`: https://github.tools.sap/AI/gen-ai-hub-sdk-js-backlog/issues/89#issuecomment-7440288